tsconfig.json

```diff
@@ -2,6 +2,9 @@
   "compileOnSave": false,
   "compilerOptions": {
       "declaration": true,
+      "lib": [
```

Comment:
Same question as the dependency updates: is it possible to break this into another PR?